source/frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -72,18 +74,36 @@ export const PropertyFilter: React.FC<React.PropsWithChildren<IPropertyFilterPro
     }
   };
 
+  const searchOptions = [
+    { label: 'PID', value: 'pid' },
+    { label: 'PIN', value: 'pin' },
+    { label: 'Address', value: 'address' },
+    { label: 'Plan #', value: 'planNumber' },
+    {
+      label: 'Historical File #',
+      value: 'historical',
+    },
+  ];
+
+  if (toggle === SearchToggleOption.Map) {
```

Comment:
this is something else we should explore when we discuss our map rework - is how to better unify or decouple the map/list search. Not part of this story of course.